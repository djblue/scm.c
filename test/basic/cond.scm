(write (cond [else 5]))
(write (cond [#t #t]))
(write
  (cond
    [#f #f]
    [#t #t]))
