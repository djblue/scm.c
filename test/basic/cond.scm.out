nil
5
#t
#t
